A test in snek5000-phill test suite fails

```raw
(venv) pierre@gre ~/tmp/snek_tmp/snek5000-phill $ pytest                                                                                            
=============================================================== test session starts ================================================================
platform linux -- Python 3.8.2, pytest-5.4.2, py-1.8.1, pluggy-0.13.1
rootdir: /home/pierre/tmp/snek_tmp/snek5000-phill
collected 3 items                                                                                                                                  

tests/test_solver.py .Fs                                                                                                                     [100%]

===================================================================== FAILURES =====================================================================
____________________________________________________________________ test_make _____________________________________________________________________

sim = <phill.solver.SimulPhill object at 0x7fa2efd80cd0>

    def test_make(sim):
>       assert sim.make.exec(["mesh", "compile"])
E       AssertionError: assert False
E        +  where False = <bound method Make.exec of <snek5000.make.Make object at 0x7fa2efd80eb0>>(['mesh', 'compile'])
E        +    where <bound method Make.exec of <snek5000.make.Make object at 0x7fa2efd80eb0>> = <snek5000.make.Make object at 0x7fa2efd80eb0>.exec
E        +      where <snek5000.make.Make object at 0x7fa2efd80eb0> = <phill.solver.SimulPhill object at 0x7fa2efd80cd0>.make

tests/test_solver.py:12: AssertionError
--------------------------------------------------------------- Captured stderr call ---------------------------------------------------------------
WARNING: Changing to shadow directory: /home/pierre/Sim_data/test/phill_run_22x16x19_V1.x1.x1._2020-05-09_17-47-44
WorkflowError in line 22 of /home/pierre/tmp/snek_tmp/snek5000-phill/src/phill/Snakefile:
Config file /home/pierre/tmp/snek_tmp/snek5000-phill/src/phill/etc/gre.yml not found.
  File "/home/pierre/tmp/snek_tmp/snek5000-phill/src/phill/Snakefile", line 22, in <module>
ERROR: WorkflowError in line 22 of /home/pierre/tmp/snek_tmp/snek5000-phill/src/phill/Snakefile:
Config file /home/pierre/tmp/snek_tmp/snek5000-phill/src/phill/etc/gre.yml not found.
  File "/home/pierre/tmp/snek_tmp/snek5000-phill/src/phill/Snakefile", line 22, in <module>
---------------------------------------------------------------- Captured log call -----------------------------------------------------------------
WARNING  snakemake.logging:logging.py:372 Changing to shadow directory: /home/pierre/Sim_data/test/phill_run_22x16x19_V1.x1.x1._2020-05-09_17-47-44
ERROR    snakemake.logging:logging.py:376 WorkflowError in line 22 of /home/pierre/tmp/snek_tmp/snek5000-phill/src/phill/Snakefile:
Config file /home/pierre/tmp/snek_tmp/snek5000-phill/src/phill/etc/gre.yml not found.
  File "/home/pierre/tmp/snek_tmp/snek5000-phill/src/phill/Snakefile", line 22, in <module>
============================================================= short test summary info ==============================================================
FAILED tests/test_solver.py::test_make - AssertionError: assert False
====================================================== 1 failed, 1 passed, 1 skipped in 0.51s ======================================================

```

--- a/src/snek5000/assets/default_configfile.yml
+++ b/src/snek5000/assets/default_configfile.yml
@@ -1,0 +1,8 @@
+CC: gcc
+FC: gfortran
+MPICC: mpicc
+MPIFC: mpif77
+MPIEXEC: "mpiexec --oversubscribe"
+CFLAGS: "-march=native"
+FFLAGS: "-march=native -mcmodel=medium -std=legacy"
+

--- a/src/snek5000/output/base.py
+++ b/src/snek5000/output/base.py
@@ -11,7 +11,7 @@ from pathlib import Path
 from socket import gethostname
 
 from fluidsim.base.output.base import OutputBase
-from snek5000 import logger, mpi
+from snek5000 import get_asset, logger, mpi
 
 
 class Output(OutputBase):
@@ -68,7 +68,17 @@ class Output(OutputBase):
                 "SNIC_RESOURCE", os.getenv("GITHUB_WORKFLOW", gethostname())
             )
         root = cls.get_root()
-        return root / "etc" / f"{host}.yml"
+        configfile = root / "etc" / f"{host}.yml"
+
+        if not configfile.exists():
+            logger.warning(
+                "Expected a configuration file describing compilers and flags: "
+                f"{configfile}"
+            )
+            configfile = Path(get_asset("default_configfile.yml"))
+            logger.info(f"Using default configuration instead: {configfile}")
+
+        return configfile
 
     def __init__(self, sim=None):
         self.sim = sim
@@ -80,6 +90,8 @@ class Output(OutputBase):
         # Same as package name __name__
         self.name_pkg = sim.info.solver.classes.Output.module_name.split(".")[0]
         self.root = self.get_root()
+        # Check configfile early
+        self.get_configfile()
 
         self._blacklist = {"prefix": "__", "suffix": (".vimrc", ".tar.gz", ".o", ".py")}
         if sim:

